metadata_manager.py

```diff
@@ -5,7 +5,7 @@
 class MetadataManager:
 
     def __init__(self):
-        self.conn = sqlite3.connect('metadata.db')
+        self.conn = sqlite3.connect('metadata.db', check_same_thread=False)
```

Comment:
We will know once we start running things concurrently haha :P

On Thu, Mar 26, 2015 at 1:55 PM, Chris Baik <notifications@github.com>
wrote:

> In metadata_manager.py
> <https://github.com/paivaspol/EECS591/pull/55#discussion_r27238344>:
>
> > @@ -5,7 +5,7 @@
> >  class MetadataManager:
> >
> >      def __init__(self):
> > -        self.conn = sqlite3.connect('metadata.db')
> > +        self.conn = sqlite3.connect('metadata.db', check_same_thread=False)
>
> eh. no opinion haha. seems fine to me
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/paivaspol/EECS591/pull/55/files#r27238344>.
>